events-application-rest-api/src/main/java/com/bounswe/bounswe2017group3/Controller/EventController.java

```diff
@@ -52,20 +53,30 @@ public EventController(EventRepository repository) {
 
         return repository.findByName(name);
     }
+    
+    //List events with respect to privacy option
+    //Example = localhost:8080/event?privacyoption=true
```

Comment:
Examples are unnecessary in the comments. But, you should add the new endpoints with their documentations to our README page here: https://github.com/bounswe/bounswe2017group3/tree/master/events-application-rest-api